pkg/node/node.go

```diff
@@ -121,6 +122,11 @@ func (node *Node) NodeGetInfo(ctx context.Context, req *csi.NodeGetInfoRequest)
 		topology[topoKey] = sasAddr
 	}
 
+	for i, sasAddr := range fcAddresses {
+		topoKey := fmt.Sprintf("%s/%s-%d", common.TopologyInitiatorPrefix, common.TopologyFCInitiatorLabel, i)
+		topology[topoKey] = sasAddr
+	}
```

Comment:
```suggestion
	for i, fcAddr := range fcAddresses {
		topoKey := fmt.Sprintf("%s/%s-%d", common.TopologyInitiatorPrefix, common.TopologyFCInitiatorLabel, i)
		topology[topoKey] = fcAddr
	}
```

Reply:
In the fmt.Sprintf("%s/%s-%d", common.TopologyInitiatorPrefix, common.TopologyFCInitiatorLabel, i), you are printing the index, nit the fcAddr, is that the intent?

Reply:
Yes that's intentional, the address is passed as the value, the index is added only to make the map key unique